django/test/utils.py

```diff
@@ -3,11 +3,16 @@
 import re
 import sys
 import time
-from unittest import skipUnless
+from unittest import skipIf, skipUnless
 import warnings
 from functools import wraps
 from xml.dom.minidom import parseString, Node
 
+try:
```

Comment:
The new import order guidelines haven't merged yet, but they call for putting try/except imports last (as isort can't parse them to order them elsewhere).